test/uk/gov/hmrc/agentfirelationship/controllers/RelationshipControllerFlagOnSpec.scala

```diff
@@ -61,9 +61,12 @@ class RelationshipControllerFlagOnSpec extends UnitSpec with MockitoSugar with B
     reset(mockMongoService, mockAuditService, mockPlayAuthConnector, mockCesaRelationship)
   }
 
-  private def authStub(returnValue: Future[~[Option[AffinityGroup], Enrolments]]) =
-    when(mockPlayAuthConnector.authorise(any(), any[Retrieval[~[Option[AffinityGroup], Enrolments]]]())(any(), any()))
-      .thenReturn(returnValue)
+//  private def authStub(returnValue: Future[~[Option[AffinityGroup], Enrolments]]): OngoingStubbing[Future[Option[AffinityGroup] ~
+//                                                                                                          Enrolments]] =
+//    when(mockPlayAuthConnector.authorise(
+//      any[Predicate](), any[Retrieval[~[Option[AffinityGroup], Enrolments]]]())
+//      (any[HeaderCarrier](), any[ExecutionContext]()))
+//      .thenReturn(returnValue)
```

Comment:
do you want to remove this commented out code?

Reply:
yep... on it

Reply:
done